features/admin.extensions.v1/configs/application.tsx

```diff
@@ -107,6 +107,8 @@ export const applicationConfig: ApplicationConfig = {
     advancedConfigurations: {
         showDefaultMyAccountApplicationEditPage: true,
         showEnableAuthorization: true,
+        showFapiBetaChip: false,
```

Comment:
Let's keep the config name as general as possible.

```suggestion
        showFapiFeatureStatusChip: false,
```

Reply:
fixed with [0a6d32d](https://github.com/wso2/identity-apps/pull/6097/commits/0a6d32ddf50aa6a2eee55308cee0f45416d9844f)